### PR TITLE
Write test for mlp-confolution-2d

### PR DESCRIPTION
**Pull request by unnonouno**
I wrote a test for `MLPConvolution2D`, and fixed a bug.

### CODE REVIEW

**Comment by okuta** [comment]
How about adding tests which check if functions use cuDNN correctly (see #894)?

**Comment by unnonouno** [comment]
I fixed it.

File: tests/chainer_tests/links_tests/connection_tests/test_mlp_convolution_2d.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+from chainer import links
+from chainer import testing
+from chainer.testing import attr
+
+
+@testing.parameterize(
+    {'use_cudnn': True},
+    {'use_cudnn': False},
+)
+class TestMLPConvolution2D(unittest.TestCase):
```
Comment (okuta) [inline tests/chainer_tests/links_tests/connection_tests/test_mlp_convolution_2d.py]: Please add `@attr.cudnn`.

Comment (unnonouno) [inline tests/chainer_tests/links_tests/connection_tests/test_mlp_convolution_2d.py]: This is a test for `__call__`. We don't need to add `@attr.cudnn`.

Comment (okuta) [inline tests/chainer_tests/links_tests/connection_tests/test_mlp_convolution_2d.py]: Sorry, it was my mistake.

**Comment by okuta** [comment]
LGTM!

**Comment by unnonouno** [comment]
This patch fixes a bug that `MLPConvolution2D` doesn't use `use_cudnn` attribute.

**Pull request merged**